ci/Jenkinsfile.qt-build

```diff
@@ -0,0 +1,224 @@
+pipeline {
+    agent {
+        label 'linux && x86_64 && qt-builder'
```

Comment:
What is `qt-builder`?

Reply:
a temporary label i set to `linux-01` to only restrict my QT6 Docker Image building experiments to that host.